website/docs/components/separator/partials/accessibility/accessibility.md

```diff
@@ -0,0 +1,15 @@
+## Conformance rating
+
+<Doc::Badge @type="success">Conformant</Doc::Badge>
+
+When used as recommended, there should not be any WCAG conformance issues with this component.
+
+## Applicable WCAG Success Criteria
+
+This section is for reference only. This component intends to conform to the following WCAG Success Criteria:
+
+<Doc::WcagList @criteriaList={{array "..." }} />
```

Comment:
@MelSumner whenever time allows, would you be able to review this from an accessibility perspective and potentially provide us with the relevant criteria? this is usually part of the CRD, but considering we don't have one for this component I thought at least to open a thread here.

Reply:
We couldn't think of relevant criteria for this component, so I removed the section.